daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/VersionTimeline.scala

```diff
@@ -55,6 +55,7 @@ private[digitalasset] object VersionTimeline {
       This(That(TransactionVersion("8"))),
       Both(This(ValueVersion("5")), LanguageVersion(LMV.V1, "6")),
       Both(This(ValueVersion("6")), LanguageVersion(LMV.V1, "7")),
+      Both(This(ValueVersion("7")), LanguageVersion(LMV.V1, "dev")),
```

Comment:
Does it need the LanguageVersion "dev" here?

Reply:
I do not want that somebody insert a line after this one before we freeze 1.8.
I add a comment about that.

Reply:
👍